### PR TITLE
:sparkles:  :memo: HomePage Partner's sections  

### LINKED ISSUE #3: 🎨 HomePage Partners Section
#  🎨 Create the partner's section on the HomePage
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a section that shows/lists all our core business partners. Check the full design below for section placement. Use the assets provided below.

partners-section

Please take note of all the design features

#### Link to full design:
```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```

#### Company Logos
VVOB:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/vvob-logo_qyvjbp.jpg
```

KSA:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/Kenya_Scouts_Association_kcoka5.svg
```

farmbetter:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/farmbetter-logo_x2acda.png
```
WWF:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/wwf-4-logo-png-transparent_tbkfcq.png
```

Enabel:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380405/elewa-group-website/client-logos/Enabel-logo_afiel7.png
```

Spotlights:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380412/elewa-group-website/client-logos/Spotlights_7_pplnvm.jpg
```

### Technical Analysis
This section should be present in the `index.html` file

### Acceptance Criteria
1. When rendered the component is pixel-perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Tugi-Ngunjiri**
# Description

Create the partner's page on the webpage 

Fixes #3 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Screenshot (optional)
Task
Screenshot from 2023-10-03 15-30-47

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,20 @@
...
...Iwd5DM323Si1UuBZF599wdfqQx5pYIGxfp9TN/Z5uM0xf7GwWnOZ0YpXdxED1n6C78/yJh2U5ON5OkAUl9M4eIrPPxKUcAsfXjbcOzbX7xC84GZQ/ew+iCU7nM8Gm36P+lBbV0Hk3wcMaKF+4q0dUpUfgtYTPH18RC7LiUe5XgNApOmFN7zjSovSnCHpr7i3gSu3xSTWvPu+cvB1onw1WC5h5D7gyhz2rpyTh1VImc+8HpVXqfIYg5GEoZjO5iTULhoj7+CFbvdkiscQ9NaEm3sv7u8sg6Q5tTR6HKNWpqb4Gbfa50NSpMfTPnxgdV0ERrhUlNnDiovwLQ3FHD349O7VJgW+ZTx9/OR+Gwg8Fpuhp7p79/7/EigO8zlTIEgAqufV8soJ5ij5/35NuJwJoYvsw7ndj+zZd5J76AuZxm4Qib08bjxvMlrJhf7gEocCv/Au/VV1uTh/7aK1kx86Wdx860Y45JuHWuIZ4emC4mlUY9L6dmT3mMNla4tUUhZsEK7qMhNkm1Xo2I3BX9yciup/6UBqK7TWrWqfg2CRhZ13gzCanMa1MdmrJbRO4fjSJfI0QdJbjLuLk2FACtYIMxuXGndRTiusFNf4LovzFTvyXsLhiMeGJZwIMSdutXYU+qsTa95jlQrj6K4aKEMjHbWqqEAVWytCeM0iRO3Wa2sIb0/pJ+C2w6nJ78rCARBT6QszuHyD+csMNmf5C6eIKG1N3L362p93pHeAMWWY2DX9Qbvh28JmXleUPIZJBvATW3fyfr79uW5z791+dJOu7J7bCS1B60crgRWLC5T/u92ObyLuJR7Y+38eEgZKfdP/jiOv32Zf1cFdiBTiX4fd4ufceg9ytyuWcCrIhOJh3SF+H6mxr5gDCsFYUSEcfz7gR7yYD8gSWo0HLjq1WscRREZsGITIjAYs4vntc4ABGDxD6uiH//jpKhy4Sw+lEGRtQ1nL55TY0sEqZmv0wpnHVL/YuHjP9msY6PpBbT8hAaNRZlYi/yn1qbloIs24ppX/3OZHjLghrHQSYqEnoj08pl3NIaAnLvpjiLvv2KMuELx6OZgagg1F3D/OVi8os7WS4JKKnr5CZrnIhblRnb1GZUVZAJdXpw8+Iax2MA9X1OBr1GGbRYwgrVR9c4jP5LgapThf8AOULfoLg3Uz8AAAAASUVORK5CYII=" alt="image">
```
Comment (JAPHETHNYARANGA) [inline index.html]: this image link is not acceptable

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
please share a screenshot of the page, web and mobile

**Comment by Tugi-Ngunjiri** [comment]
> Please share a screenshot of the page, web, and mobile
Screenshot from 2023-10-03 15-30-47
Screenshot from 2023-10-03 15-38-52

Screenshot from 2023-10-03 16-04-11

File: index.html (hunk 1 above)
Comment (Tugi-Ngunjiri) [reply index.html]: > this image
Screenshot from 2023-10-04 12-21-03
ge link is not acceptable

**Comment by Tugi-Ngunjiri** [comment]
> Please share a screenshot of the page, web, and mobile
Screenshot from 2023-10-03 15-38-52
Screenshot from 2023-10-03 16-04-11

**Comment by Tugi-Ngunjiri** [comment]
> Please share a screens
Screenshot from 2023-10-03 15-38-52
hot of the page, web, and mobile

Screenshot from 2023-10-03 16-04-11

**Comment by Tugi-Ngunjiri** [comment]
> Please share a screenshot of the page, web, and mobile
 Mobile;
Screenshot from 2023-10-04 13-22-35
   Web ;
Screenshot from 2023-10-04 13-23-56

**Comment by Tugi-Ngunjiri** [comment]
I worked on this issue with   https://github.com/Lord-Gitonga

**Review by JAPHETHNYARANGA — APPROVED** [review]
Good attempt

**Pull request merged**